data/input/oneshot_itokawa_opengl.json

```diff
@@ -77,13 +64,25 @@
                 "file": "data/models/itokawa_f3145728.obj",
                 "name": "Itokawa"
             },
-            "albedo": 0.15,
```

Comment:
Is it intended that the albedo value is deleted?

Reply:
Dropping the albedo was intended as it's covered by the bidirectional reflectance distribution function parameters `brdf_params`. Hapke 2012 BRDF is the default model currently for OpenGL and there's a parameter there called single scattering albedo, which has a similar, although more fine-grained meaning.

Reply:
Alright, just wanted to be sure